package.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
+<package format="1" xmlns="https://wiki.freecad.org/Package_Metadata">
+  <name>Ondsel Lens</name>
+  <description>Workspace Manager for Ondsel and Local Workspaces</description>
+  <version>0.1.0</version>
+  <date>2023-07-06</date>
+  <maintainer email="development@ondsel.com">Shai Seger</maintainer>

```

Comment:
Is maintainer name **Shai Seger** intentional?

Reply:
Nope.  Oversight.  Fixed.